com.chartboost.helium/Editor/HeliumIntegrationChecker.cs

```diff
@@ -0,0 +1,335 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.PackageManager.UI;
+using UnityEngine;
+using PackageInfo = UnityEditor.PackageManager.PackageInfo;
+using ImportOptions = UnityEditor.PackageManager.UI.Sample.ImportOptions;
+
+namespace Editor
+{
+    public class HeliumIntegrationChecker
+    {
+        private const string UnityAds = "UnityAds";
+        private const string Helium = "Helium";
+        private const string HeliumWindowTitle = "Helium Unity SDK - Integration Status Checker";
+        private const string HeliumPackageName = "com.chartboost.helium";
+        private const string HeliumSamplesInAssets = "Assets/Samples/Helium SDK";
+        private const string UnityAdsPackageName = "com.unity.ads";
+        private const string UnityAdsUncommentWindow = "unity-ads-uncomment-window";
+        private static readonly string HeliumSamplesMetaInAssets = $"{HeliumSamplesInAssets}.meta";
+        private static readonly Version HeliumUnityAdsSupportedVersion = new Version(4, 2, 1);
+        private static readonly string UnityAdsSDKCommented = $"<!-- <androidPackage spec=\"com.unity3d.ads:unity-ads:{HeliumUnityAdsSupportedVersion}\"/> -->";
```

Comment:
Is there a way to get the package name from unity ads? If they change package name in a future (which is unlikely), this will need updating too.

```suggestion
        private static readonly string UnityAdsSDKCommented = $"<!-- <androidPackage spec=\"com.unity3d.ads:unity-ads:{HeliumUnityAdsSupportedVersion}\"/> -->";
        
      
```